pgbot/utils/embed_utils.py

```diff
@@ -538,7 +538,12 @@ def __init__(
         self,
         message: discord.Message,
         pages: list[discord.Embed],
-        caller: Optional[discord.Member] = None,
+        caller: Optional[
```

Comment:
We should not take `discord.User` here. The main reason being, `PagedEmbed` is broken on DMs, so we cannot have this on DMs, and in other cases, we will have `discord.Member` object at hand anyways